ghp_import/__init__.py

```diff
@@ -1,5 +1,3 @@
-#! /usr/bin/env python
```

Comment:
Rather than renaming `ghp-import → ghp_import/__init__.py` it would probably be better to just do `ghp-import → ghp_import.py`.